src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +44,48 @@
      * Андрей - 600
      * София - 900</p>
      */
+    private static final DateTimeFormatter FORMAT = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        LocalDate fromDate = LocalDate.parse(dateFrom, FORMAT);
+        LocalDate toDate = LocalDate.parse(dateTo, FORMAT);
+        if (fromDate.isAfter(toDate)) {
+            throw new IllegalDateParametersException("Wrong parameters");
+        }
+        int[] salaries = new int[names.length];
```

Comment:
try to solve without this array and another method to work with it

Reply:
Redundant array was removed